src/Scaler.Demo/OrderGenerator/Program.cs

```
@@ -14,79 +14,40 @@ internal static class Program
     {
         private static CosmosDbConfig _cosmosDbConfig;
 
+        public static string _applicationName = "cosmosdb-order-generator";
+
         public static async Task Main(string[] args)
         {
-            if (args.Length != 1 || !new[] { "generate", "setup", "teardown" }.Contains(args[0]))
-            {
-                Console.WriteLine();
-                Console.WriteLine("Please use one of the following verbs with the command:");
-                Console.WriteLine("    generate : Add new orders to the order-container");
-                Console.WriteLine("    setup    : Create Cosmos database and order-container");
-                Console.WriteLine("    teardown : Delete Cosmos database and containers inside");
-                Console.WriteLine();
-                return;
-            }
-
             // _cosmosDbConfig should be initialized once the host is built.
             Host.CreateDefaultBuilder(args)
                 .ConfigureAppConfiguration(builder => _cosmosDbConfig = CosmosDbConfig.Create(builder.Build()))
                 .Build();
 
-            switch (args[0])
+            switch (_cosmosDbConfig.Action)
             {
-                case "generate": await GenerateAsync(); break;
-                case "setup": await SetupAsync(); break;
-                case "teardown": await TeardownAsync(); break;
+                case "setup":
+                    await SetupAsync();
+                    break;
+                case "teardown":
+                    await TeardownAsync();
+                    break;
+                case "generate":
+                    await CreateOrdersAsync(_cosmosDbConfig.OrderCount, _cosmosDbConfig.IsSingleArticle);
+                    break;
+                default:
+                    Console.WriteLine("Invalid action. Valid actions are: setup, teardown, generate");
+                    break;
```

should we throw an exception here?